app/styles/globals.scss

```diff
@@ -27,13 +28,34 @@ body {
 }
 
 .page-container {
-  max-width: 600px;
+  max-width: 1000px;
 }
 
 .container-data {
   background: var(--background-tint);
 }
 
+.doses-data-grid {
+  display: grid;
+  grid-template-columns: repeat(12, minmax(0, 1fr));
+  gap: 1.25rem;
+
+  & .doses-data-grid-item {
+    background: var(--background-tint);
+    border-radius: $radius-large;
+
+    grid-column: span 12 / span 12;
+
+    @media (min-width: 640px) {
+      grid-column: span 6 / span 6;
+    }
+
+    @media (min-width: 1024px) {
```

Comment:
Will extract these into breakpoints 🙏 - might do in a separate PR, because it would be good to have a wider convo about what break points we have